client/views/profile.html

```diff
@@ -21,15 +21,17 @@ <h2>
             <td class="field">{{ field }}</td>
             <td ng-hide="editStatus" ng-style="(profileUser[key]) ? {} : {'font-style':'italic'}">
               {{ profileUser[key] || 'none' }}
+              <input ng-show="key==='resume' && profileUser.resume === none" type="button" 
+                        onclick="location.href='https://google.com';" value="upload resume here"/>
```

Comment:
can you find a way to set the link for the button in `.env` instead of hard-coding it here?